app/src/main/res/raw/muser_informed_consent.html

```diff
@@ -0,0 +1,75 @@
+<p align="left">
+	<strong>Project ID</strong>
+	# 2020-xxxx
+</p>
+<p align="left">
+	Researchers at the University of South Florida (USF) study many topics. To do this, we need the
```

Comment:
Quick question - did you take a look at this in-app? The lines are a bit longer here than in the previous version - not sure if this matters.

Reply:
Yes, I looked at the consent in the app. I personally don't find any difference as opposed to just scrolling more than usual. Here's the screenshot of how it would look:-

<img src = "https://user-images.githubusercontent.com/22369029/98019085-59629200-1dcf-11eb-9045-e45272ae9d74.png" width = 300/>

Reply:
Could you give this a try without adding the newlines in the file (i.e., it being one long line for each section)? It looks like there is an awkward extra space between "this" and "research" in the first paragraph - I'm wondering if that would remove it.

Reply:
oh yeah. There was a space between the two words, I removed them and it sort of fixed it. Would you still want me to make the paragraphs one line?

Reply:
If that fixed it, then you can leave as-is - it's easier to read/edit as-is